### PR TITLE
Migrate secondary message/envelope actions to material icons

### PR DESCRIPTION
**Pull request by ChristophWurst**
Ref https://github.com/nextcloud/groupware/issues/38

### CODE REVIEW

**Comment by JuliaKirschenheuter** [comment]
Thank you for fixing!

Should we also change icons in actions inside of EnvelopePrimaryActions?

**Comment by ChristophWurst** [comment]
Yes. They just need some styling fixes.

File: src/components/Envelope.vue [hunk 1]
```diff
@@ -85,39 +85,62 @@
 				</ActionButton>
 			</EnvelopePrimaryActions>
 			<template v-if="!moreActionsOpen">
-				<ActionButton icon="icon-junk"
-					:close-after-click="true"
+				<ActionButton :close-after-click="true"
 					@click.prevent="onToggleJunk">
+					<template #icon>
+						<AlertOctagonIcon
```
Comment (GretaD) [inline src/components/Envelope.vue:91]: this doesnt work, when i mark as spam, the icon is not shown

**Comment by GretaD** [comment]
Please notice on the top right the small arrow showing up when i click select or mark as spam
https://user-images.githubusercontent.com/12728974/169501814-8c302aa3-dd1d-4dd7-bdfe-0df7bfee5606.mp4

File: src/components/Envelope.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/components/Envelope.vue:91]: The action icon  renders for spam and non-spam message. Clicking toggles the state. Then I can toggle back. This works in the envelope list and in the message view.

What exactly does not show for you?

**Comment by ChristophWurst** [comment]
> Please notice on the top left the small arrow showing up when i click select or mark as spam

Have you compared this to main? I think it's caused by the popover menu.

File: src/components/Envelope.vue (hunk 1 above)
Comment (JuliaKirschenheuter) [reply src/components/Envelope.vue:91]: for me works, besides icons inside of EnvelopePrimaryActions

Comment (GretaD) [reply src/components/Envelope.vue:91]: Sorry, i was testing this wrong, as i said on my first comment, when i toggle mark as span, the icon doesnt show on the avatar (not related to this pr). 
What you have done here is changing the left icon on the menu.

**Review by GretaD — APPROVED** [review]
all the icons are shown correctly

**Review by JuliaKirschenheuter — APPROVED** [review]

**Pull request merged**